packages/grant-explorer/src/context/RoundContext.tsx

```diff
@@ -93,7 +93,7 @@ function fetchRoundsById(
   dispatch({ type: ActionType.SET_LOADING, payload: true });
 
   dataLayer
-    .getLegacyRoundById({
```

Comment:
I think we aren't using this anymore right ? Maybe we can delete the query itself ?